### PR TITLE
[Feature][Field] Url

### PR DESCRIPTION
**Pull request by AhmedBG**
I tried adding a new prop => linkText, to show a custom a link text in the Cell, I was not able too 😞

### CODE REVIEW

File: src/url/index.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { BaseListTypeInfo, FieldTypeFunc } from "@keystone-6/core/types"
+import { TextFieldConfig } from "@keystone-6/core/dist/declarations/src/fields/types/text"
+import { text } from "@keystone-6/core/fields"
+
+const urlRegex =
```
Comment (DiesIrae) [inline src/url/index.ts:6]: source in comment please

**Review by DiesIrae — APPROVED** [review]
Add import to src/index.ts :-)

**Comment by AhmedBG** [comment]
@DiesIrae don't I also need to up the package version ?

**Comment by DiesIrae** [comment]
> @DiesIrae don't I also need to up the package version ?

Yes! I had written this in my previous review but it got lost and I forgot to rewrite it!

**Pull request merged**